subprojects/smoke-test/src/smokeTest/groovy/org/gradle/smoketests/ThirdPartyPluginsSmokeTest.groovy

```diff
@@ -148,7 +155,7 @@ class ThirdPartyPluginsSmokeTest extends AbstractSmokeTest {
                     mavenCentral()
                 }
                 dependencies {
-                    classpath('org.springframework.boot:spring-boot-gradle-plugin:1.4.0.RELEASE')
+                    classpath('org.springframework.boot:spring-boot-gradle-plugin:1.5.2.RELEASE')
```

Comment:
Maybe we should try out https://spring.io/blog/2017/04/05/spring-boot-s-new-gradle-plugin as well. We could potentially had a Spock `where` clause to pass in different versions.